.github/workflows/test-integration.yml

```diff
@@ -29,6 +29,9 @@ on:
       - ".github/workflows/test-integration.yml"
       - "!docs/**"
 
+env:
+  DAGGER_CACHE_BASE: dagger-ci-integration
```

Comment:
some workflows are setting `DAGGER_LOG_FORMAT` and some are not -- maybe standardize one or the other?

Reply:
Some are set it in `helpers.bash`, I think I can take care of that in another PR